umbraco/Umbraco-CMS
# v9: Created new system information section in help panel

# Notes
- Created a new System Information section, within the "help" section
- Created a new UserData Model
- Created IUserDataService Interface
- Created UserDataService class and registered the service
- Created Unit Tests to test the service
- Created Cypress test to System Information displayed

# How to test
- Navigate to the client project and run `npm install` & `npm run build`
- Run the project
- Open the help section and the System Information and see that all values display correctly
- Try updating your .NET 5 to a newer version and see if it registers the change
- Try changing your users language and see if it registers the change(You have to refresh the page after changes)
- Try going to the backoffice from a different browser
- Try deleting your browser of choice from `platform.service.js` (either with `npm run dev` already running, or rebuilding the client project with `npm install` & `npm run build`
